source/two_phase_base.cc

```diff
@@ -1097,6 +1101,8 @@ compute_bubble_statistics(std::vector<Tensor<2,3> > *,
     data[3+d] = global_velocity[d]/global_volume;
   for (unsigned int d=0; d<dim; ++d)
     data[3+dim+d] = global_mass_center[d]/global_volume;
+  data[3+dim+dim] = (std::pow(pi, 1./3.)
+		     * std::pow(6*global_volume, 2./3.)) / (global_area);
```

Comment:
I first thought that you would write into invalid memory when simply writing into position `[3+dim+dim]` (you did not change the size of the vector), but then I realized that data gets size `4+2*dim`. So it simply held the value zero before this patch, right?